core/templates/pages/exploration-player-page/services/exploration-engine.service.ts

```diff
@@ -488,7 +492,7 @@ export class ExplorationEngineService {
      }]);
      if (questionHtml === null) {
        this.answerIsBeingProcessed = false;
-       this.alertsService.addWarning('Expression parsing error.');
+       this.alertsService.addWarning('Question name should not be empty.');
```

Comment:
If think this should be "question content" (question name is confusing). But I don't understand why we are refering to questions in the "exploration" engine service in the first place. It looks like it is referring to the state content. Can you add a TODO to handle this separately (i.e. we should not be referring to questions in this service)?

Reply:
done!